src/message/mailbox/serde.rs

```diff
@@ -198,7 +198,7 @@ mod test {
             from_str(r#""yin@dtb.com, Hei <hei@dtb.com>, Kai <kayo@example.com>""#).unwrap();
         assert_eq!(
             m,
-            "<yin@dtb.com>, Hei <hei@dtb.com>, Kai <kayo@example.com>"
```

Comment:
Not sure to understand why a nameless address should be parsed with angles. The new parser seems to parse it correctly.

Reply:
I believe that both syntaxes are correct and need to be supported.

```
mailbox         =       name-addr / addr-spec
name-addr       =       [display-name] angle-addr
angle-addr      =       [CFWS] "<" addr-spec ">" [CFWS] / obs-angle-addr
```

https://datatracker.ietf.org/doc/html/rfc2822#section-3.4

`mailbox` and be a `name-addr` which can be just an `angle-addr`.

Reply:
Indeed you are right, I will adjust tests and the parser to match those cases.

Reply:
In fact it already works as expected, with and without angles :heavy_check_mark: